paymentsheet/src/test/java/com/stripe/android/utils/CompletableSingle.kt

```diff
@@ -0,0 +1,18 @@
+package com.stripe.android.utils
+
+import com.stripe.android.common.coroutines.Single
+import kotlinx.coroutines.CompletableDeferred
+
+internal class CompletableSingle<T>(val value: T? = null) : Single<T> {
```

Comment:
Should we add some unit tests for this?

Reply:
This is actually a test class. I didn't expose it in `src` since we don't have a use case for `CompletableSingle` outside of converting it from a `Flow`.